### PR TITLE
Add audit logs documentation

### PR DESCRIPTION
**Pull request by sean1588**
Document audit logs feature. I'm not completely sure where on the site this should live. Right now I have it under the user guides section but I am open to suggestions otherwise.

Issue: #2604

### CODE REVIEW

File: content/docs/guides/auditing/_index.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+meta_desc: "This page is a guide on how to audit your organization's infrastructure as code activity"
+menu:
+  userguides:
+    identifier: auditing
+    weight: 2
+---
+
+## Overview
+
+Audit logs enable you to track the activity of users within an
```
Comment (spara) [inline content/docs/guides/auditing/_index.md:12]:
```suggestion
Audit logs tracks the activity of users within an
```

File: content/docs/guides/auditing/_index.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+organization. They attempt to answer what a user did, when
```
Comment (spara) [inline content/docs/guides/auditing/_index.md:13]:
```suggestion
organization. They record what a user did, when
```

File: content/docs/guides/auditing/_index.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+they did it and where. They help answer these questions
```
Comment (spara) [inline content/docs/guides/auditing/_index.md:14]:
```suggestion
they did it and where. They answer these questions
```

File: content/docs/guides/auditing/_index.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+by recording user actions.
+
+Pulumi's audit logs allow you to account for the activity your
```
Comment (spara) [inline content/docs/guides/auditing/_index.md:17]:
```suggestion
Pulumi's audit logs let you account for user activity
```

File: content/docs/guides/auditing/_index.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+users are taking within your organization. These logs are available to
```
Comment (spara) [inline content/docs/guides/auditing/_index.md:18]:
```suggestion
within your organization. These logs are available to
```

File: content/docs/guides/auditing/_index.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+organizations with an Enterprise level subscription. The logs are immutable and
+and record all user actions. Auditing makes the activity
+of members in an organization attributable.
+The logs capture the UNIX timestamp of the event, the user
+who invoked the action, the event that took place, and the source IP
+of the call the user made.
+
```
Comment (spara) [inline content/docs/guides/auditing/_index.md:25]: Add a sample of a log, e.g.,

```
timestamp, user, event, ip
```

File: content/docs/guides/auditing/_index.md [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+## Viewing Audit Logs in the Console
+
+Audit logs are available to organizations with an Enterprise level subscription
+only. If you are an organization administrator, you can view your organization's audit logs,
```
Comment (spara) [inline content/docs/guides/auditing/_index.md:29]:
```suggestion
only. Organization administrators can view your organization's audit logs,
```

File: content/docs/guides/auditing/_index.md [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+by selecting your organization from the organization drop down. Then click on the settings tab.
```
Comment (spara) [inline content/docs/guides/auditing/_index.md:30]:
```suggestion
by selecting your organization from the organization dropdown and clicking on the settings tab.
```

File: content/docs/guides/auditing/_index.md [hunk 9]
```diff
@@ -0,0 +1,71 @@
...
+On the left nav-bar you should see a tab called Audit Logs. Clicking here will allow you to view
```
Comment (spara) [inline content/docs/guides/auditing/_index.md:31]:
```suggestion
On the left nav-bar is the Audit Logs tab, clicking on the tab displays
```

File: content/docs/guides/auditing/_index.md [hunk 10]
```diff
@@ -0,0 +1,71 @@
...
+the most recent audit logs for your organization.
+
+![auditlogs](./auditlogs.png)
+
+This will show the most recent events in decending order. You can
```
Comment (spara) [inline content/docs/guides/auditing/_index.md:36]:
```suggestion
This displays the most recent events in descending order. You can
```

File: content/docs/guides/auditing/_index.md [hunk 11]
```diff
@@ -0,0 +1,71 @@
...
+also filter logs by a particular user by clicking on the user's avatar. Doing so will
```
Comment (spara) [inline content/docs/guides/auditing/_index.md:37]:
```suggestion
also filter logs by user by selecting the user's avatar. Doing so will
```

File: content/docs/guides/auditing/_index.md [hunk 12]
```diff
@@ -0,0 +1,71 @@
...
+filter out the events performed by the user you selected.
```
Comment (spara) [inline content/docs/guides/auditing/_index.md:38]:
```suggestion
filter out the events performed by the selected user.
```

File: content/docs/guides/auditing/_index.md [hunk 13]
```diff
@@ -0,0 +1,71 @@
...
+
+Audit logs can also be exported to a downloadable csv format. The logs can be exported through the UI Console
```
Comment (spara) [inline content/docs/guides/auditing/_index.md:40]:
```suggestion
Audit logs can be exported to a downloadable CSV format through the UI Console
```

File: content/docs/guides/auditing/_index.md [hunk 14]
```diff
@@ -0,0 +1,71 @@
...
+by clicking on the `DOWNLOAD` button in the upper left hand corner of the audit logs view.
+
+## Exporting Audit Logs through the API
+
+The audit logs can be exported through the API using the following endpoint. A `startTime`
```
Comment (spara) [inline content/docs/guides/auditing/_index.md:45]:
```suggestion
The audit logs can be exported through the API. A `startTime`
```

File: content/docs/guides/auditing/_index.md [hunk 15]
```diff
@@ -0,0 +1,71 @@
...
+query parameter must be passed and is used to query audit records _before_ the specified
```
Comment (spara) [inline content/docs/guides/auditing/_index.md:46]:
```suggestion
query parameter must be passed to query audit records _before_ the specified
```

File: content/docs/guides/auditing/_index.md [hunk 16]
```diff
@@ -0,0 +1,71 @@
...
+`startTime` (UNIX timestamp).
+
+```
+GET https://api.pulumi.com/api/orgs/${org}/auditlogs/export?startTime=${time}
+```
+
+Optionally, a user to filter by can also be specified as a query parameter to filter audit logs
```
Comment (spara) [inline content/docs/guides/auditing/_index.md:53]:
```suggestion
Optionally, a user can also be specified as a query parameter to filter audit logs
```

**Comment by lukehoban** [comment]
> I'm not completely sure where on the site this should live. Right now I have it under the user guides section but I am open to suggestions otherwise.

I think for now it's probably best under Pulumi Console.  

This page should likely have the same banner at the top as the self-hosted page to call out that it is Enterprise only.

Also - probably best for the title for the current content to be "Audit Logs" which matches the feature itself.

**Pull request merged**